clippy_lints/src/macro_use.rs

```diff
@@ -142,7 +143,7 @@ impl<'tcx> LateLintPass<'tcx> for MacroUseImports {
         }
     }
     fn check_crate_post(&mut self, cx: &LateContext<'_>) {
-        let mut used = FxHashMap::default();
+        let mut used = BTreeMap::new();
```

Comment:
Is `BTreeMap` faster for this than `FxIndexMap`? If so there are some cases where this should be used instead, but I'm unsure if this is the case..

Reply:
Could go either way and would vary based on the size of the map/complexity of the keys. An index map would work as well but I didn't pick one over the other for any particular reason